subsys/bluetooth/services/fast_pair/use_case/Kconfig

```diff
@@ -75,6 +76,10 @@ config BT_FAST_PAIR_USE_CASE_MOUSE
 endchoice
 
 # Collection of unsupported Fast Pair features that can be selected by the use case Kconfig.
+config BT_FAST_PAIR_USE_CASE_UNSUPPORTED_BOND_MANAGER
```

Comment:
This seems to belong to the previous commit (in general the first 2 commits seem a bit mixed up now)

Reply:
I've merged those first two commits. They're interdependent anyway.